docs/en/plan-deployment/cost.md

```diff
@@ -42,20 +42,35 @@ Key assumptions include:
-|                         |   7  |  122   | MSK (m5.large * 2, connector MCU * 1)   |       417      |   3  |     549   |
-|                         | 7    |  122   | None              |             |  3    |      132   |
-|100RPS           |   43  |  122  |     Kinesis(On-demand)              |      86       |  3   |     254 |
-|                         | 43    |   122  | Kinesis (Provisioned 2 shard)   |      26       | 3    |     194  |
-|           |   43  |  122  |     MSK (m5.large * 2, connector MCU * 1)              |      417       |  3   |     585
-|           |   43  |  122  |     None              |             |  3    |     168
-|1000RPS           |   396  |   122 |     Kinesis(On-demand)              |      576       |  14   |    1108 |
-|                         |  396   |  122   | Kinesis (Provisioned 10 shard)   |    146         |   14  |     678  |
-|           |  396   | 122   |     MSK (m5.large * 2, connector MCU * 2~3)              |      530       |  14  |     1062
-|           |  396   | 122   |     None              |            |  14   |     532
+| Request Per Second | ALB | EC2  |  Buffer type      | Buffer cost | S3   |  Total (USD/Month) |
+| ------------------ | --- | ---  |  --------------   | ----------- | ---  |  --------- |
+| 10RPS              |  7  |  122 |  Kinesis (On-Demand) |    36       |   3  |     168  |
+|                    |  7  |  122 |  Kinesis (Provisioned 2 shard)   |      22       |  3   |   154  |
+|                    |  7  |  122 |  MSK (m5.large * 2, connector MCU * 1)   |       417      |   3  |     549   |
+|                         | 7    |  122 |  None              |             |  3    |      132   |
+|100RPS           |   43  |  122  |  Kinesis(On-demand)              |      86       |  3   |     254 |
+|                         | 43    |   122 |  Kinesis (Provisioned 2 shard)   |      26       | 3    |     194  |
+|           |   43  |  122  |   MSK (m5.large * 2, connector MCU * 1)              |      417       |  3   |     585
+|           |   43  |  122 |      None              |             |  3    |     168
+|1000RPS           |   396  |   122 |      Kinesis(On-demand)              |      576       |  14   |    1108 |
+|                         |  396   |  122  |  Kinesis (Provisioned 10 shard)   |    146         |   14  |     678  |
+|           |  396   | 122  |      MSK (m5.large * 2, connector MCU * 2~3)              |      530       |  14  |     1062
+|           |  396   | 122   |      None              |            |  14   |     532
+
+## Data transfer
+When data is transferred from EC2 to the downstream data sink, there are associated costs. Below is the example of data transfer costs based on 1000 RPS and 1KB payload of a request.
+
+1. EC2 Network In: This does not incur any costs.
+2. EC2 Network Out: There are three data sink options:
+    - S3: Recommended to use VPC endpoint, which does not incur any costs.
```

Comment:
It's not accurate. No cost only when using the S3 gateway, not the VPC interface endpoint.

Reply:
changed to S3 Gateway endpoints